examples/s3_access.ipynb

```diff
@@ -82,24 +98,40 @@
     "\n",
     "import boto3\n",
     "\n",
-    "s3 = boto3.client('s3', **creds)\n",
-    "uri = 's3://harmony-uat-staging/public/harmony/netcdf-to-zarr/817a3e99-d53c-4169-b9f1-82cc947793be/2020_01_01_7f00ff_global.zarr'\n",
-    "bucket_name = uri.split('/')[2]\n",
-    "object_name = '/'.join(uri.split('/')[3:])\n",
-    "file_name = uri.split('/')[-1]\n",
+    "def parse_s3_url(url):\n",
```

Comment:
I think I made a note of this on the other ticket that had S3 work, but rather than require every use to adapt the URL to what works with boto3, I think we should default to returning a tuple of (bucket_name, object_name) or something similarly easy to drop into boto3